aspnetpatch-21/test.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "aspnetpatch-21",
+  "version": "2.1",
+  "versionSpecific": true,
```

Comment:
Is this for all of 2.1 or just 2.1.400? You can also use feature numbers (since like yesterday)
And do we want it tested only and only on this version, or any other future versions as well? (specific)

Reply:
The test verifies things work on all 2.1 sdks.
We'll want to test a similar thing on 2.2 sdks. For now, the test is 2.1 specific.

Reply:
Okay, that gives me an idea - would having a $1 be the version, i.e. 2.1 help parametrize this test?

Reply:
> would having a $1 be the version, i.e. 2.1 help parametrize this test?

Yes. It would be more informative to pass it as environment variables, e.g. DOTNET_TEST_VERSION is the runtime version.

Let's scope this PR to testing it on 2.1. You can explore how it can be made more generic for 2.2.

Reply:
Yup